packages/orchestration/src/exos/orchestrator.js

```diff
@@ -111,27 +112,23 @@ export const prepareOrchestratorKit = (
         },
       },
       orchestrator: {
-        /** @type {Orchestrator['getChain']} */
+        /** @type {PromiseToVow<Orchestrator['getChain']>} */
         getChain(name) {
           if (name === 'agoric') {
-            // XXX when() until membrane
-            return when(
-              watch(
-                chainHub.getChainInfo('agoric'),
-                this.facets.makeLocalChainFacadeWatcher,
-              ),
+            // @ts-expect-error Type 'Vow<Voidless>' is not assignable to type 'Vow<Chain<any>>'.
```

Comment:
Should there be a TODO/bug to fix these types?

Reply:
Yes, I added [cb5f043](https://github.com/Agoric/agoric-sdk/pull/9591/commits/cb5f043976d33e1c922388d29683c4e72ec81077) which clearly spells out the compromises I made on types. I suspect the `PromiseVow` added here doesn't get us all the way with generics needed for `getChain`. 

For the ticket, I am attributing it is part of the remaining work for #9449.